parser/Parser/Parser.py

```diff
@@ -0,0 +1,174 @@
+from neo4j import GraphDatabase
+import re,unicodedata,unidecode
+
+
+class Parser:
+
+    def __init__(self,filename,uri="bolt://localhost:7687"):
+        self.driver = GraphDatabase.driver(uri)
+        self.session = self.driver.session() #Doesn't create error even if there is no active database
+        self.filename=filename
+
+    def file_name(self):
+        """open the file filename and return the list of entries, each entry is a list a the sanitized lines"""
+        self.filename = self.filename + ( '.txt' if (len(self.filename)<4 or self.filename[-4:]!=".txt") else '')  #to not get error if extension is missing
+
+    def file_iter(self,start=0):
+        """generator to get the file line by line"""
+        counter=0
+        with open(self.filename,"r",encoding="utf8") as file:
+            for line in file:
+                if counter < start : counter+=1
+                else :
+                    # sanitizing
+                    #remove any space characters at end of line
+                    line = line.rstrip()
+                    #replace ’ (typographique quote) to simple quote '
+                    line = line.replace("’", "'")
+                    #replace commas that have no space around by a lower comma character and do the same for escaped comma (preceded by a \) (to distinguish them from commas acting as tags separators)
```

Comment:
In the case of longer comments, try to reduce the length or do use the triple quote notation to make it readable.
```suggestion
                    """ replace commas that have no space around by a lower comma character and do the same for escaped comma (preceded by a \) (to distinguish them from commas acting as tags separators) """
```